curriculum/challenges/english/07-scientific-computing-with-python/learn-classes-and-objects-by-building-a-sudoku-solver/6606bf4561f8794a0d345919.md

```diff
@@ -0,0 +1,57 @@
+---
+id: 6606bf4561f8794a0d345919
+title: Step 27
+challengeType: 20
+dashedName: step-27
+---
+
+# --description--
+
+It's time to test the `valid_in_row` method. Call `valid_in_row` on `gameboard`. Pass it `0` and `8` as the arguments and print the result.
```

Comment:
This would be a good place to tell why only 2 parameters instead of 3 are being passed to `valid_in_row`.

Reply:
Yeah, absolutely :+1: